scripts/gulpfile.factory.js

```diff
@@ -134,29 +134,38 @@ module.exports = function factory({
   );
 
   const getURL = (string, type) => {
-    const re = new RegExp(`get\\s+${type}Url\\([^)]*\\)\\s*{\\s*return\\s+"([^"]+)"`, "g");
+    const re = new RegExp("get\\s+" + type + "Url\\([^)]*\\)\\s*{\\s*return\\s+\"([^\"]+)\"", "g");
     const parse = re.exec(string);
-    return typeof parse === "object" && parse !== null ? parse[1] : null;
+
+    if (Array.isArray(parse) && parse !== null && parse.length > 0) {
+      return parse[1];
+    }
+
+    return null;
   };
 
   const fetchHtml = url => {
+    if (!url) return null;
+
     // Check for the html template
-    if (url && fs.existsSync(path.join(paths.source, url))) {
+    if (fs.existsSync(path.join(paths.source, url))) {
       // Returns a string with the cleaned up HTML
-      return decomment(
-        fs
+      // decomment(
```

Comment:
Was this to get something to work or is it more for better dev experience?

Reply:
Yikes, I don't even remember.  let me add it back and see what breaks

Reply:
Oh I remember now!  It's stripping out comments and that breaks the commented out code in the pfe-jump-links.js file

Reply:
👍